### PR TITLE
Fixed PS3 class to PS5 class | clan3columns Layout and other views/php

### PR DESCRIPTION
**Pull request by Slipi85**
Fixed PS3 class to PS5 class

unter user/panel/profile lässt sich bei geburtstag der Kalender nicht öffnen.

### CODE REVIEW

File: application/modules/birthday/views/index/index.php [hunk 1]
```diff
@@ -14,11 +14,12 @@
             </span>
         </div>
         <?php foreach ($this->get('birthdayListNOW') as $birthdaylist): ?>
-            <div class="card-body">
+            <div class="card-body row">
                 <div class="col-xl-12">
+                  <div class="row"
```
Comment (blackcoder87) [inline application/modules/birthday/views/index/index.php:19]: Indentation is off. Check your editor settings.

1 tab = 4 spaces.

File: application/modules/media/views/admin/iframe/index.php [hunk 2]
```diff
@@ -32,7 +32,6 @@
             <div class="row">
             <?php foreach ($this->get('medias') as $media): ?>
                 <?php if (in_array($media->getEnding(), explode(' ',$this->get('media_ext_video')))): ?>
-                    <div class="row">
```
Comment (blackcoder87) [inline application/modules/media/views/admin/iframe/index.php:35]: I think I saw adding and removal of this a few times now.
Indentation is off with that div removed.

File: static/css/ilch.css [hunk 3]
```diff
@@ -105,3 +105,19 @@ img {
   margin:40px 0 20px;
   border-bottom:1px solid #eee
 }
+
+/* .pagination */
+
```
Comment (blackcoder87) [inline static/css/ilch.css:110]: Remove empty line.

**Review by blackcoder87 — CHANGES_REQUESTED** [review]

**Comment by blackcoder87** [comment]
Improve the title of the PR. Just "Fixed" is bad.

File: application/layouts/clan3columns/index.php [hunk 4]
```diff
@@ -10,7 +10,7 @@
     </head>
     <body>
         <header>
-          <nav class="navbar navbar-expand-lg navbar-light navbar-gaming bg-light fixed-top d-sm-block d-md-none">
+          <nav class="navbar navbar-expand-lg navbar-dark navbar-gaming fixed-top d-sm-block d-md-none">
```
[the remark this answers is not in the document]
Comment (Slipi85) [reply application/layouts/clan3columns/index.php:13]: Ok, will change it

File: application/layouts/clan3columns/index_full.php [hunk 5]
```diff
@@ -10,7 +10,7 @@
     </head>
     <body>
       <header>
-        <nav class="navbar navbar-expand-lg navbar-light navbar-gaming bg-light fixed-top d-sm-block d-md-none">
+        <nav class="navbar navbar-expand-lg navbar-dark navbar-gaming fixed-top d-sm-block d-md-none">
```
[the remark this answers is not in the document]
Comment (Slipi85) [reply application/layouts/clan3columns/index_full.php:13]: Ok, will change it

File: application/modules/media/views/admin/iframe/index.php (hunk 2 above)
Comment (Slipi85) [reply application/modules/media/views/admin/iframe/index.php:35]: That's correct. Under the Media/iframe module, the row class was included in the loop in all php.

File: application/modules/birthday/views/index/index.php (hunk 1 above)
Comment (Slipi85) [reply application/modules/birthday/views/index/index.php:19]: Its fixed

File: static/css/ilch.css (hunk 3 above)
Comment (Slipi85) [reply static/css/ilch.css:110]: its fixed

**Pull request merged**